deploy/dev-gcp.yml

```diff
@@ -62,24 +62,6 @@ spec:
       # Tillater innlogging for alle brukere.
       allowAllUsers: true
       claims:
-        # Kun de gruppene brukeren er medlem av vil bli lagt til i claims på tokenet.
-        groups:
```

Comment:
Hvorfor fjerne groups?
Den hindrer brukere som ikke er medlem av disse gruppene fra å logge inn.

Reply:
Har lagt inn igjen groups for veileder, saksbehandler,drift. Og krever at man må være medlem av minst en av disse for å få logget inn